classic/create.go

```diff
@@ -234,15 +234,34 @@ func customizeClassicChroot() error {
 		return fmt.Errorf("failed to cleanup classic /run dir: %s (%s)", err, output)
 	}
 
+	// Add SUDO_USER to sudo group in the chroot to ensure that
+	// sudo work for the calling user.
+	sudoUser := os.Getenv("SUDO_USER")
+	if sudoUser != "" {
+		// We need to use "runInClassicEnv" so that we get the
+		// bind mount of the /var/lib/extrausers directory.
+		// Without that the "SUDO_USER" will not exist in the chroot
+		if err := runInClassicEnv("usermod", "-a", "-G", "sudo", sudoUser); err != nil {
+			return fmt.Errorf("failed to add %s to the sudo users: %s", sudoUser, err)
+		}
+	}
+
```

Comment:
@niemeyer On older installs there is indeed the "admin" group instead of the (newer) "sudo" group. This is not be a problem for us because we control how the snappy image is created. Currently this is the default user permission: http://bazaar.launchpad.net/~ubuntu-core-dev/livecd-rootfs/trunk/view/head:/live-build/ubuntu-core/hooks/01-setup_user.chroot (adm, sudo). If people add users manually we will honor the groups of those users too this way on install of classic.